### ISSUE TITLE
row names missing when using df_paste/tribble_paste

### ISSUE THREAD

**Issue by sowla** [MilesMcBain/datapasta#87]
Hi, thank you for your awesome package!! :D

I think row names are not preserved when using df_paste and tribble_paste, but I'm not sure if I'm using them wrong..?

``` r
datapasta::df_paste(mtcars[1:3, 1:3])
#> data.frame(
#>          mpg = c(21, 21, 22.8),
#>          cyl = c(6, 6, 4),
#>         disp = c(160, 160, 108)
#> )
```
<sup>Created on 2019-02-20 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

``` r
datapasta::tribble_paste(mtcars[1:3, 1:3])
#> tibble::tribble(
#>   ~mpg, ~cyl, ~disp,
#>     21,    6,   160,
#>     21,    6,   160,
#>   22.8,    4,   108
#>   )
```
<sup>Created on 2019-02-20 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by jonocarroll** [comment]
The lack of rownames in `df_paste` was, I suspect, largely due to thinking mainly about the use-case of clipboard data, but it shouldn't be terribly difficult to include the rownames if they're present to begin with. `tibble` doesn't support rownames so they shouldn't be expected there. Are you interested in making a PR for this? Otherwise I'll have a go.

**Comment by sowla** [comment]
Hi, sorry about the late reply!

Oops, I forgot that about tibbles! I think I would like to have a go, if it's not too urgent (I'm very new and I'm not sure how long I'd take)..?

**Comment by sowla** [comment]
Hi again, I submitted a [PR](https://github.com/MilesMcBain/datapasta/pull/88) for this, but it went faster than I expected, so I'm not sure if I missed something.. I also don't know if I broke the edits up in a preferable way, so please let me know if I should change anything!

Thanks in advance!

**Comment by MilesMcBain** [comment]
merged in #88
